packages/react-form-state/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [0.3.2]
```

Comment:
I'm guessing there is a 0.3.1 coming first?

Reply:
yeah 0.3.1 is actually already deployed, changelog entry is just missing

Reply:
I'll fix that separately